src/ControlSurface/Controls/Colorpicker/Colorpicker.qml

```diff
@@ -0,0 +1,185 @@
+//  A toy QML colorpicker control, by Ruslan Shestopalyuk
+import QtQuick 2.11
+import QtQuick.Layouts 1.1
+import QtQuick.Controls 2.4
+
+//import "content"
+
+Rectangle {
+    id: picker
+    property color colorValue: paletteMode ? _rgb(paletts.paletts_color,
+                                          alphaSlider.value) : _hsla(
+                                         hueSlider.value,
+                                         sbPicker.saturation,
+                                         sbPicker.brightness,
+                                         1)
+    property bool enableAlphaChannel: true
+    property bool enableDetails: true
+    property int colorHandleRadius: 8
+    property bool paletteMode: false
+    property bool enablePaletteMode: false
+    property string switchToColorPickerString: "Palette..."
```

Comment:
In a future commit (really not urgent, maybe add it to the kanban board): put the text strings in qsTr() to allow to provide translations further down the line

Reply:
Also a good practice is to mark such "constant" strings as readonly: 

    readonly property string switchToColorPickerString: "Palette..."

(likewise for any other constants). Again, not for that PR, but let's do it later